tools/metrics/histograms/metadata/cobalt/histograms.xml

```diff
@@ -275,7 +275,7 @@ Always run the pretty print utility on this file after editing:
   </summary>
 </histogram>
 
-<histogram name="Cobalt.Media.SbPlayer.Create.Timing" units="microseconds"
+<histogram name="Cobalt.Media.SbPlayer.Create.Timing.2" units="microseconds"
```

Comment:
Would some variation of ".version2" be clearer that this is a new iteration?

Reply:
Changed to `LatencyTiming` as Joel suggested.